content/doc/book/pipeline/index.adoc

```diff
@@ -6,43 +6,152 @@ layout: chapter
+Jenkins, supporting use-cases that span from simple continuous integration to
+comprehensive continuous delivery pipelines. By modeling a series of related
+tasks, users can take advantage of the many features of Pipeline:
+
+* *Code*: Pipelines are implemented "as code," and typically checked into
+  source control, giving teams the abilit to edit, review, and iterate upon
+  their delivery pipeline.
+* *Durable*: Pipelines can survive both planned, and unplanned, restarts of the
+  Jenkins master.
+* *Pausable*: Pipelines can optionally stop and wait for human input or approval
+  before continuing the Pipeline run.
+* *Versatile*: Pipelines support complex real-world continuous delivery
+  requirements, including the ability to fork/join, loop, and perform work in
+  parallel.
+* *Extensible*: The Pipeline plugin supports custom extensions to its DSL
+  footnoteref:[dsl]
+  and multiple options for integration with other plugins.
+
+
+While Jenkins has always allowed rudimentary forms of chaining Freestyle Jobs
+together to perform sequential tasks,
+footnote:[Additional plugins have been used to implement complex behaviors
+utilizing Freestyle Jobs such as the Copy Artifact, Parameterized Trigger,
+and Promoted Builds plugins]
+Pipeline makes this concept a first-class citizen in Jenkins.
+
+Building on the core Jenkins value of extensibility, Pipeline is also
+extensible both by users with <<pipeline/shared-libraries#,Pipeline Shared Libraries>>
+and by plugin developers.
+footnoteref:[ghof,link:https://plugins.jenkins.io/github-organization-folder[GitHub
+Organization Folder plugin]]
+
+
+The flowchart below is an example of one continuous delivery scenario easily
+modeled in Jenkins Pipeline:
+
+image::/images/pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]
+
+
+[[terms]]
+== Pipeline Terms
 
-If you are a Jenkins administrator and want to know more about managing Jenkins nodes and instances, see
-<<managing#,Managing Jenkins>>.
+[[step]]
+Step::
+    A single task; fundamentally steps tell Jenkins _what_ to do. In
+    order to execute the shell command `make`, the `sh`
+    step would be used. For example `sh 'make'`.
+    When a plugin extends the Pipeline DSL, that typically means the plugin has
+    implemented a new _step_.
 
-If you are a system administrator and want learn how to back-up, restore, maintain as Jenkins servers and nodes, see
-<<operating#,Operating Jenkins>>.
+[[node]]
+Node::
+    Generally speaking, all _work_ which is done by a Pipeline, will be done in
+    the context of a single, or multiple, `node` step declarations, which does two things:
+    . Schedules the steps contained within the block to run by adding an item
+      to the Jenkins queue. As soon as an executor is free on a node, the
+      steps will run.
+    . Creates a workspace, a directory specific to that particular
+      Pipeline, where files can be checked out from source control and work can
+      be done.
```

Comment:
Tightening this up:

s/ . Creates a workspace, a directory specific to that particular Pipeline, where files can be checked out from source control and work can be done./
 . Creates a workspace (a directory specific to that particular Pipeline) where you can work on files checked out from source control.